pkg/interface/src/views/apps/chat/components/chat-editor.js

```diff
@@ -142,19 +165,38 @@ export default class ChatEditor extends Component {
         paddingTop='8px'
         width='calc(100% - 88px)'
         className={inCodeMode ? 'chat code' : 'chat'}
+        color="black"
       >
-        <CodeEditor
+        {MOBILE_BROWSER_REGEX.test(navigator.userAgent)
```

Comment:
That said this will basically mean that iPad doesn't have the native HTML input lol. Thanks Apple for making iOS Safari vs. Desktop Safari illegible